### ISSUE TITLE
Player Dimension NBT not updated for 1.16

### ISSUE THREAD

**Issue by NeunEinser** [SPGoding/datapack-language-server#737]
DPH+ still suggests integer values (-1, 0, 1) for player dimension data. In 1.16, it's a namespaced id though.

I'm not sure if the issue lies in the nbtdoc this time, as it looks kinda correct there
https://github.com/Yurihaia/mc-nbtdoc/blob/74d84a498d34bfcac934a6a619b5f1bdf9ce1305/minecraft/entity/mob/player.nbtdoc#L7-L8

So either the definition of `id(minecraft:dimension)` is outdated (wherever that may be), or DPH+ does not understand that datatype properly for 1.16.

**Comment by SPGoding** [comment]
Is the `datapack.env.dataVersion` 1.16 or 1.16.2? The correct type only exists in the nbtdoc for 1.16.2 and after.

**Comment by NeunEinser** [comment]
is cmd version 1.16 fine? dataversion is set tp 1.16.2 for me.
![grafik](https://user-images.githubusercontent.com/12124394/98610008-ec7b5a80-22ee-11eb-8318-7304e1a7f65f.png)

**Comment by SPGoding** [comment]
This is weird. Perhaps you could try deleting `%globalStoragePath%/1.16.2/Nbtdoc.json` because if you launched VS Code after 1.16.2 was released but before Yurihaia released the nbtdoc for 1.16.2, DHP might end up using the nbtdoc for 1.16.1.

**Comment by NeunEinser** [comment]
There is no `Nbtdoc.json` 🤔 
![grafik](https://user-images.githubusercontent.com/12124394/98700254-496f2300-2378-11eb-929a-32be0bd7fa65.png)

**Comment by NeunEinser** [comment]
I found this in the log:
```
[VanillaData: Nbtdoc for 1.16.2] Fetching from GitHub https://raw.githubusercontent.com/Yurihaia/mc-nbtdoc/1.16.2-gen/build/generated.json...
[VanillaData: Nbtdoc for 1.16.2] SyntaxError: Unexpected token : in JSON at position 3 (1/3)
[VanillaData: Nbtdoc for 1.16.2] Used the fallback.
```

**Comment by SPGoding** [comment]
🤦 Seems like Yurihaia didn't release the tag for 1.16.2. This will be fixed when #663 is resolved.

**Comment by SPGoding** [comment]
Duplicate of #663
